kedro/config/common.py

```diff
@@ -63,6 +67,7 @@ def _get_config_from_patterns(
 
     config = {}  # type: Dict[str, Any]
     processed_files = set()  # type: Set[Path]
+    custom_anyconfig_args = anyconfig_args if anyconfig_args else DEFAULT_ANYCONFIG_ARGS
```

Comment:
```suggestion
    custom_anyconfig_args = anyconfig_args or DEFAULT_ANYCONFIG_ARGS
```

Reply:
You can't do this because it's `Optional` and may be `None`